azext_iot/tests/iothub/core/test_iothub_storage_int.py

```diff
@@ -376,3 +398,35 @@ def test_user_identity_storage(self):
             )
 
         self.tearDown()
+
+    def wait_till_job_completion(self, job_id):
+        tries = 0
+        status = self.cmd(
+            f"iot hub job show -n {self.entity_name} -g {self.entity_rg} --job-id {job_id}"
+        ).get_output_in_json()["status"]
+
+        while status not in ["failed", "completed"] and tries < SETUP_MAX_ATTEMPTS:
+            status = self.cmd(
+                f"iot hub job show -n {self.entity_name} -g {self.entity_rg} --job-id {job_id}"
+            ).get_output_in_json()["status"]
+            print(f"Try number: {tries} for {self.entity_name}: status {status}")
+            sleep(10)
+            tries += 1
+
+        if status == "failed":
+            print(
```

Comment:
What is the purpose of having a print here?

Reply:
To log the specific error response (if we need to bother the service again) - will change it to a logger.error.